src/component/timeBlock/template/TimeBox.tsx

```diff
@@ -0,0 +1,65 @@
+import { styled } from 'styled-components';
+import { ErrorMessage } from '@/component/timeBlock/atoms/ErrorMessage';
+import {
+  TimeBlockGroup,
+  TimeList,
+  SelectAllButton,
+} from '@/component/timeBlock/molecules';
+import { TimeBlockHeader } from '@/component/timeBlock/organisms';
+import { useState } from 'react';
+
+export const TimeBox = () => {
+  const [nowPage, setNowPage] = useState(1);
+  const [activeDate, setActiveDate] = useState<boolean[]>([]);
+
+  return (
+    <>
+      <TimeBlockHeader
+        page={nowPage}
+        onSetPage={setNowPage}
+        activeDate={activeDate}
+      />
+      <TimeBoxBlock>
+        <TimeList start="9:00" end="17:00" />
+        <TimeBlockGroup page={nowPage} onSetActiveDate={setActiveDate} />
+      </TimeBoxBlock>
+      {activeDate.includes(false) ? (
+        <ErrorMessage>시간이 선택되지 않은 날짜가 있습니다</ErrorMessage>
+      ) : null}
```

Comment:
이런 상황이라면 3항연산자가 아니라 && 면 될거같습니다

```suggestion
      {activeDate.includes(false) && (
        <ErrorMessage>시간이 선택되지 않은 날짜가 있습니다</ErrorMessage>
      )}
```

Reply:
근데 이거 에러 메세지만 보고 하는 얘기긴 한데.. 특정 날짜에 내가 그냥 바빠서 안되면 선택하지 않을 수도 있지 않나요..?